frontend/openchat-service-worker/src/service_worker.ts

```diff
@@ -263,51 +264,27 @@ async function showNotification(n: Notification, id: string): Promise<void> {
     }
 
     const path = notificationPath(n);
-    let tag: string | undefined = undefined;
 
     if (isMessageNotification(n)) {
         if (icon === undefined && n.messageType === "File") {
             icon = FILE_ICON;
         }
-    } else {
-        tag = path;
     }
 
-    const existing = await self.registration.getNotifications();
-    const matching = existing.filter((n) => n.data.path === path);
-    const closed = closeExcessNotifications(matching, MAX_NOTIFICATIONS_PER_GROUP);
```

Comment:
By avoiding closing existing notifications I'm hoping this will fix the issue I intermittently see where it says "This site was updated in the background" at the same time as it shows a normal notification.